fdbserver/worker.actor.cpp

```diff
@@ -540,7 +548,8 @@ ACTOR Future<Void> workerServer( Reference<ClusterConnectionFile> connFile, Refe
 	state WorkerCache<InitializeStorageReply> storageCache;
 	state Reference<AsyncVar<ServerDBInfo>> dbInfo( new AsyncVar<ServerDBInfo>(ServerDBInfo()) );
 	state Future<Void> metricsLogger;
-	state std::map<KeyValueStoreType::StoreType, std::pair<Future<Void>, PromiseStream<InitializeTLogRequest>>> sharedLogs;
+	state std::map<std::pair<KeyValueStoreType::StoreType, TLogSpillType::SpillType>,
```

Comment:
this should be by version rather than spill type, so that future tlog formats can use different file names and be dispatched easily